template/provision/templates/application.yml.template

```diff
@@ -0,0 +1,12 @@
+# These environment variables are applied to all environments
+SECRET_KEY_BASE: {{secret_key_base}}
```

Comment:
yes, and it is -- this pulls from the ansible variables in the vault which can be defined differently for different environents